docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+services:
+  app:
+    build: .
+    volumes:
+      - .:/dockermount
+    ports:
+      - 8000:8000
+    entrypoint: ["python", "/dockermount/docker_django_management.py"]
+    command: ["python", "manage.py", "runserver", "0.0.0.0:8000"]
+    links:
+      - db
+    environment:
+      - DDM_IS_RUNNING_IN_DOCKER=yup
+      - DATABASE_URL=postgres://usaspender@db/usaspending
+      - DDM_HOST_USER=usaspender
+      - PYTHONUNBUFFERED=yup
+  db:
+    image: postgres:9.5.4
+    volumes:
+      - pgdata:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_DB=usaspending
+      - POSTGRES_USER=usaspender
+volumes:
+  pgdata:
```

Comment:
The official postgres container already uses an unnamed [data volume](https://docs.docker.com/engine/tutorials/dockervolumes/) for its storage, but here we give it a name so that it's easy to do things with. For instance, `docker volume rm usaspendingapi_pgdata` will delete it, which can be useful if you ever want to blow away your development db.

(All Docker Compose containers and volumes are prefixed with a slugified form of your project's root directory name, so if you have this project checked out to a directory called `usaspending-api`, all of Docker Compose's stuff will be prefixed with `usaspendingapi_` when examining it through the `docker` CLI tool.)